src/assets/templates/seo-bounty-cards.hbs

```diff
@@ -1,197 +1,197 @@
+            font-size: 33px;
+            font-weight: 500;
+            line-height: 120%;
+            color: rgba(255, 255, 255, 0.4);
+        }
+
+        .title {
+            font-weight: 500;
+            font-size: 48px;
+            line-height: 120%;
+            color: #ffffff;
+
+            overflow: hidden;
+            text-overflow: ellipsis;
+            display: -webkit-box;
+            -webkit-line-clamp: 2;
+            line-clamp: 2;
+            word-wrap: break-word;
+            -webkit-box-orient: vertical;
+        }
+
+        .repo-tag {
+            font-weight: 700;
+            font-size: 25.6px;
+            line-height: 120%;
+            letter-spacing: 0.1em;
+            text-transform: uppercase;
+            color: #4250e4;
+
+            border: #4250e4 solid 2px;
+            border-radius: 8px;
+            padding: 6px 10px;
+            max-width: fit-content;
+        }
+
+        .amount {
+            font-size: 70px;
+            line-height: 120%;
+            display: flex;
+            gap: 20px;
+            align-items: center;
+            font-weight: 500;
+            color: #ffffff;
+            margin: 1rem 0;
+        }
+
+        .currency {
+            margin-right: 10px;
+            color: #4250e4;
+            font-size: 40px;
+            line-height: 120%;
+            font-weight: 500;
+            text-transform: uppercase;
+            font-feature-settings: "tnum" on, "lnum" on, "ss02" on, "ss03" on,
+                "ss04" on;
+        }
+
+        .container-legend {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            justify-content: space-between;
+        }
+
+        .container-legend h3 {
+            font-weight: 600;
+            font-size: 38px;
+            line-height: 100%;
+            color: #ffffff;
+            margin: 0.75rem 0;
+        }
+
+        .container-legend span {
+            font-weight: 500;
+            font-size: 30px;
+            line-height: 100%;
+            color: rgba(255, 255, 255, 0.4);
+            text-transform: uppercase;
+        }
+    </style>
+    <div class="container">
+        <div class="container-status">
+            <div id="status-label-{{state}}" class="status-label"><span>{{state}}</span></div>
+            <span class="issue-number">#{{id}}</span>
+        </div>
+        <h1 class="title">
+            {{title}}
+        </h1>
+        {{#if isFudingBounty}}
+            <h1 class="amount">{{fundedAmount}} <span class="title">of</span> {{fundingAmount}}<span class="currency">{{currency}}</span></h1>
+        {{else}}
+            <h1 class="amount">{{amount}} <span class="currency">{{currency}}</span></h1>
+        {{/if}}
+        <div class="container-legend">
+            <div>
+                <h3>{{working}}</h3>
+                <span>WORKING</span>
+            </div>
+            <div>
+                <h3>{{deliverables}}</h3>
```

Comment:
why is the whole file marked as changed when you only changed this?? This can't continue to happen. Make sure you spacing is the same as unix-style spacing, check your settings, check your editor settings, etc.. Anything, except whole files where what changes is _spacing_ and two letters. There's not enough sanity in the world for this.